content/en/real_user_monitoring/platform/connect_rum_and_traces.md

```diff
@@ -114,9 +114,21 @@ To start sending just your iOS application's traces to Datadog, see [iOS Trace C
 
 **Note**: `traceSampleRate` **does not** impact RUM sessions sampling. Only backend traces are sampled out.
 
+4. _(Optional)_ If you set a `traceSampleRate`, to ensure backend services' sampling decisions are still applied, configure the `traceContextInjection` initialization parameter to `sampled` (set to `all` by default).
+
+_Example_: If you set the `traceSampleRate` to 20% in the Browser SDK:
+- When `traceContextInjection` is set to `all`, **20%** of backend traces are kept and **80%** of backend traces are dropped.
+
+{{< img src="real_user_monitoring/connect_rum_and_traces/traceContextInjection_all.png" alt="traceContextInjection set to all" style="width:90%;">}}
+
+- When `traceContextInjection` is set to `sampled`, **20%** of backend traces are kept. For the remaining **80%**, the browser SDK **does not inject** a sampling decision. The decision is made on the server side, based on the tracing library head-based sampling [configuration][2] (backend sample rate set to 40% in the example below).
```

Comment:
```suggestion
  - When `traceContextInjection` is set to `sampled`, **20%** of backend traces are kept. For the remaining **80%**, the browser SDK **does not inject** a sampling decision. The decision is made on the server side and is based on the tracing library head-based sampling [configuration][2]. In the example below, the backend sample rate is set to 40%, and therefore 32% of the remaining backend traces are kept.
```

Reply:
Hi @dussault-antoine , I added the bit at the end to clarify the example further, but not sure if it is accurate. Can you verify and edit if needed? Thank you!